condor_watch_q.py

```diff
@@ -216,6 +223,24 @@ def parse_args():
     return args
 
 
+def parse_unknown_args(unknown):
+
+    for unknown_arg in unknown:
+        print("Unknown command.", end=" ")
+        if unknown_arg.isdigit():
+            print("Did you mean condor_watch_q -clusters {} ?".format(unknown_arg))
+        elif "-totals" in unknown_arg:
+            print("Did you mean condor_watch_q -no-table?")
```

Comment:
Make sure there's a space after the suggested command in all of these suggestions.

Reply:
Sorry, I was a little unclear: the space should go after the command but before the `?`, to make sure that it's easy to copy-paste the suggestion.